### ISSUE TITLE
Sharing UI to public in addition to localhost

### ISSUE THREAD

**Issue by zubairahmed-ai** [imartinez/privateGPT#1365]
Hi @lopagela 

Thanks for a great repo, although I still couldn't figure out how to run it on a GPU but that's another issue, can you kindly tell me how to share the gradio instance publicly with others `without` auth?

Would appreciate an urgent reply.

Thanks

**Comment by Error-418** [comment]
You need to get your external IP address from a service like whatismyip.com, give that to whomever you're trying to have connect to you along with the port number, then unblock their connection attempts through your firewall.
